pages/about/policies/AUP.html

```diff
@@ -0,0 +1,143 @@
+<DOCTYPE! html>
+<p>&nbsp;</p>
+<p><u><strong>Mount Sion C.B.S.</strong></u></p>
+<p>&nbsp;</p>
+<p><u><strong>School Internet Acceptable Usage Policy.</strong></u></p>
+<p><u><strong>November 2018.</strong></u></p>
+<p><br /> <u><strong>Rationale.</strong></u></p>
+<p>&nbsp;</p>
+<p>The aim of this acceptable usage policy (AUP) is to ensure that students get maximum benefit from the learning opportunities offered by the school&rsquo;s ICT and internet resources in a safe and responsible manner. The school has two fully equipped computer rooms, one portable class set of twenty four chromebooks, all classrooms have a teacher&rsquo;s computer with wired internet access and all classrooms and common areas have WIFI access.The school&rsquo;s wifi system provides two networks, an open public network and a password protected private network. Internet use and access is considered a school resource and privilege. If the school&rsquo;s AUP is not adhered to, this privilege will be withdrawn and appropriate sanctions will be imposed.</p>
+<p>&nbsp;</p>
+<p><strong>School Acceptable Use Strategy.</strong></p>
+<p>&nbsp;</p>
+<p>The school employs a number of strategies in order to maximise learning opportunities and reduce risks associated with the internet. These strategies include:</p>
+<p>&nbsp;</p>
+<p><strong>General</strong></p>
+<ul>
+<li>
+<p>Open internet sessions will always be supervised by a teacher.</p>
+</li>
+<li>
+<p>Filtering systems (HEANET) and or equivalent systems will be used where appropriate in order to minimise the risk of exposure to inappropriate material.</p>
+</li>
+<li>
+<p>The school will regularly monitor students&rsquo; internet usage.</p>
+</li>
+<li>
+<p>Each student will be issued with a username and password for their own network account. Students must use their own account at all times.</p>
+</li>
+<li>
```

Comment:
@evanhearne @SinenWew some of the additional html tags here are causing the format of the text to have a lot of spacing. We saw this with previous ones as well. To make the layout look a bit better could you make changes to improve this?

Reply:
@laurafitzgerald that isn't a problem. Me and @SinenWew will get to work on that on Wednesday hopefully.